Ktranslate crashing after commit from #788 - `panic: runtime error`

Customer doesn't have an IPv6 interface on their host and is using `ping_only` devices in their config file. Following commit from #788, container crashes with this error on start-up:

```golang
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1091282]
```

--- a/pkg/inputs/snmp/ping/kaping/pinger.go
+++ b/pkg/inputs/snmp/ping/kaping/pinger.go
@@ -31,6 +31,8 @@ func NewPinger(cfg Config, addrs []netip.Addr) (*Pinger, error) {
 	if err != nil {
 		if has4 {
 			return nil, fmt.Errorf("IPv4 socket: %w", err)
+		} else {
+			sock4 = nil
 		}
 	}
 
@@ -38,6 +40,8 @@ func NewPinger(cfg Config, addrs []netip.Addr) (*Pinger, error) {
 	if err != nil {
 		if has6 {
 			return nil, fmt.Errorf("IPv6 socket: %w", err)
+		} else {
+			sock6 = nil
 		}
 	}
 

